netket/graph/lattice.py

```diff
@@ -20,14 +20,68 @@
 import networkx as _nx
 import warnings
 from typing import Tuple, Union, Optional
+from math import pi
+from itertools import product
+from dataclasses import dataclass
 
-cutoff_tol = 1e-5
+from netket.utils.semigroup import Identity, Element
+from .symmetry import SymmGroup
+from netket.utils import HashableArray
+
+tol_digits = 5
```

Comment:
It's confusing to have two quantities that are botgh 'editable'. We should stick with one.
If it makes sense to edit `tol_digits` instead of `cutoff_tol` due to implementation details, i'm okay with switching to that but then the other one should disappear or be made internal (prepend an underscore).